content/applications/finance/payment_acquirers/stripe.rst

```diff
@@ -39,14 +69,23 @@ To retrieve the publishable and secret keys, follow this `link to your API keys
 Webhook Signing Secret
 ~~~~~~~~~~~~~~~~~~~~~~
 
-To retrieve the webhook signing secret, you first need to create a webhook.
+To retrieve the webhook signing secret, create a webhook either automatically or manually.
+
+Create the webhook automatically
+********************************
+
+Make sure your :ref:`Publishable and Secret keys <stripe/api_keys>` are filled in, then click on the
+*Generate your Webhook* button.
+
+Create the webhook manually
+***************************
 
-To do so, follow this `link to your webhooks <https://dashboard.stripe.com/webhooks>`_, or log into
-your Stripe dashboard and go to :menuselection:`Developers --> Webhooks`. Then, click on **Add
-endpoint** in your **Hosted endpoints**. A form opens, where you'll need to add the following data:
+Follow this `link to your webhooks <https://dashboard.stripe.com/webhooks>`_, or log into your
```

Comment:
Better to have the link on a text about the link rather than on the word "link".

For example, Open you [link] (webhooks on Stripe)

Reply:
I agree but (review tip) notice that this diff is not linked to the PR. It was simply moved and adapted to the change of context ;)

Reply:
okééééééééé